daffodil-core/src/test/scala/org/apache/daffodil/core/xml/TestXMLLoaderWithLocation.scala

```diff
@@ -34,7 +35,7 @@ class TestXMLLoaderWithLocation {
     // if it can.
     val testXML = <xs:schema xmlns:xs={XMLUtils.XSD_NAMESPACE}><xs:annotation/></xs:schema>
     try {
-      using(new java.io.FileWriter(tmpXMLFileName)) { fw =>
+      Using(new java.io.FileWriter(tmpXMLFileName)) { fw =>
```

Comment:
Same with these `Using`'s, do these want to be `Using.resource`? These are unlikely to fail, but if they do we want to make sure the exception is thrown so the test fails and doesn't hide the exception. I imagine most uses of the `Using.apply`'s want to be `Using.resource` unless we are doing something with the resulting `Try`, otherwise we're just ignoring an exception.